packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -12,6 +12,7 @@ export interface FeaturesData {
   postEngagementNonClickable?: boolean;
   postModalByDefault?: boolean;
   postCardVersion?: string;
+  postCardShareVersion?: string;
```

Comment:
I think this could be the reason why the build fails. Kindly add the import after this change as well.
```suggestion
  postCardShareVersion?: ShareVersion;
```